src/pkg/cli/compose/normalize.go

```diff
@@ -0,0 +1,15 @@
+package compose
+
+import (
+	"regexp"
+	"strings"
+)
+
+var (
+	nonAlphanumeric = regexp.MustCompile(`[^a-zA-Z0-9]+`)
+)
+
+func NormalizeServiceName(s string) string {
+	// TODO: replace with the code from compose-go
```

Comment:
@edwardrf what did you use again?

Reply:
I don't think compose go does service name normalization in compose go, only [project name normalization](https://github.com/compose-spec/compose-go/blob/9d0d133e13d0955e27520c6317d08822b7c5de5f/loader/loader.go#L661) which removes all non allowed characters and '_' and '-' from beginning of the string.